app/routes/session.py

```diff
@@ -177,6 +177,7 @@ def _set_questionnaire_supplementary_data(
         dataset_id=new_sds_dataset_id,
         identifier=identifier,  # type: ignore
         survey_id=metadata["survey_id"],  # type: ignore
+        sds_schema_version=schema.json.get("sds_schema_version", None),
```

Comment:
As `.get()` implicitly returns `None` if the key isn't found, might not need to override the output here?